### PR TITLE
[Cloud Security] adding readme for cspm & kspm policy templates

### PR DESCRIPTION
**Pull request by tinnytintin10**
A PR to only provide the content I want us to use for the KSPM and CSPM policy templates. 

cc @oren-zohar @kfirpeled @tehilashn

### CODE REVIEW

**Comment by kfirpeled** [comment]
@tinnytintin10 can you replace the description and names inside manifest.yml as well? we need to remove that placeholder over there

**Comment by tinnytintin10** [comment]
@kfirpeled @oren-zohar can you please review and merge? thanks!

File: packages/cloud_security_posture/docs/README.md [hunk 1]
```diff
@@ -1,207 +1,35 @@
...
+- Networking objects that allow ingress to remote server administration ports (22, 3389, etc.)
 
-For in-depth, step-by-step instructions to help you get started with KSPM, please read through [our getting started guide](https://ela.st/getting-started-with-kspm). 
+And much more! For a complete overview of CSPM, including step-by-step getting started guidance, check out [CSPM's documentation](https://ela.st/cspm)
 
-## Using KSPM  
+## Kubernetes Security Posture Management (KSPM)
 
-After you install this integration, the pages described in the table below will begin to get populated with security posture data. Please read the ["Use Cases"](https://ela.st/kspm-use-cases-8-5) section of the KSPM documentation for step-by-step instructions on how to use these pages to get insight into and improve the security posture of your Kubernetes clusters. 
+KSPM discovers and evaluates the components that make up your Kubernetes cluster against hardening guidelines defined by the Center for Internet Security (CIS) to help you identify and remediate configurations risks like:
```
Comment (oren-zohar) [inline packages/cloud_security_posture/docs/README.md:17]: maybe link the CIS site?

Comment (oren-zohar) [inline packages/cloud_security_posture/docs/README.md:17]:
```suggestion
KSPM discovers and evaluates the components that make up your Kubernetes cluster against hardening guidelines defined by the [Center for Internet Security](https://www.cisecurity.org/) (CIS) to help you identify and remediate configurations risks like:
```

File: packages/cloud_security_posture/docs/README.md [hunk 2]
```diff
@@ -1,207 +1,35 @@
...
-Access keys are long-term credentials for an IAM user or the AWS account root user.
-To use access keys as credentials, you need to provide:
-
-* `Access Key ID`: The first part of the access key.
-* `Secret Access Key`: The second part of the access key.
-
-For more details refer to [AWS Access Keys and Secret Access Keys](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys).
-
-#### Use temporary security credentials
+- Kubelete servers that allow anonymous Auth
```
Comment (oren-zohar) [inline packages/cloud_security_posture/docs/README.md:19]:
```suggestion
- Kubelete servers that allow anonymous auth
```

File: packages/cloud_security_posture/docs/README.md [hunk 3]
```diff
@@ -1,207 +1,35 @@
...
+- Unencrypted traffic to load balancers
+- Pod Security Policies (PSPs) that enable the admission of privileged containers
```
Comment (oren-zohar) [inline packages/cloud_security_posture/docs/README.md:21]: PSPs are deprecated, maybe we can a different example

File: packages/cloud_security_posture/docs/README.md [hunk 4]
```diff
@@ -1,207 +1,35 @@
...
-```
-
-## Leader election
-
-To collect cluster level data (compared to node level information) the integration makes use of the [leader election](https://www.elastic.co/guide/en/fleet/master/kubernetes_leaderelection-provider.html) mechanism.
-This mechanism assures that the cluster-level data is collected by only one of the agents running as a part of the DaemonSet and not all.
-
-Cluster level data example: List of the running pods.
-Node level data example: kubelet configuration.
+After deploying the CSPM and/or KSPM integration, the pages described in the table below will begin to get populated with security posture data. Please read the respective use cases section for [CSPM](https://ela.st/cspm-use-cases) and [KSPM](https://ela.st/kspm-use-cases) for step-by-step instructions on how to use these pages to get insight into and improve your cloud security posture.
```
Comment (oren-zohar) [inline packages/cloud_security_posture/docs/README.md:27]: implies that we support them both at the same time?

**Review by oren-zohar — CHANGES_REQUESTED** [review]

File: packages/cloud_security_posture/docs/README.md (hunk 4 above)
Comment (tinnytintin10) [reply packages/cloud_security_posture/docs/README.md]: Updated the paragraph to be a bit more clear. 

https://github.com/elastic/integrations/pull/5133/commits/6749cff839ca439d77885bd0a39adce5f866a789

File: packages/cloud_security_posture/docs/README.md (hunk 3 above)
Comment (tinnytintin10) [reply packages/cloud_security_posture/docs/README.md]: used a different example:

https://github.com/elastic/integrations/pull/5133/commits/6d09e28196bc3e52d384f4511cd3a42e9c6a54b5

File: packages/cloud_security_posture/docs/README.md [hunk 5]
```diff
@@ -1,207 +1,35 @@
...
 
+CSPM discovers and evaluates the services in your cloud environment, like storage, compute, IAM, and more, against hardening guidelines defined by the Center for Internet Security (CIS) to help you identify and remediate configurations risks like:
 
-## Getting started guide
+- Publicly exposed storage buckets 
+- IAM Users without MFA enabled 
+- Networking objects that allow ingress to remote server administration ports (22, 3389, etc.)
 
-For in-depth, step-by-step instructions to help you get started with KSPM, please read through [our getting started guide](https://ela.st/getting-started-with-kspm). 
+And much more! For a complete overview of CSPM, including step-by-step getting started guidance, check out [CSPM's documentation](https://ela.st/cspm)
```
Comment (bmorelli25) [inline packages/cloud_security_posture/docs/README.md:13]:
```suggestion
And much more! For a complete overview of CSPM, including step-by-step getting started guidance, check out [CSPM's documentation](https://ela.st/cspm).
```

**Pull request merged**